WebService/ComparisonEngineUI/ComparisonEngineUI/Views/BarsPage.xaml

```diff
@@ -1,10 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:behaviorsPack="clr-namespace:Xamarin.Forms.BehaviorsPack;assembly=Xamarin.Forms.BehaviorsPack"
              x:Class="ComparisonEngineUI.Views.BarsPage">
     <ContentPage.Content>
         <StackLayout>
-            <SearchBar Placeholder="Search Bars..." />
+            <SearchBar x:Name="searchBar"
```

Comment:
It would be better if there was a TextColor property, so that color would be the same for everyone.

Reply:
Does the color depend on the device? The text color is black on my phone.

Reply:
it does, if I remember correctly, Aurimas textcolor is white